<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>238</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>426</width>
    <height>238</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>426</width>
    <height>238</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>智慧小区登录系统</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>111</width>
            <height>91</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>111</width>
            <height>91</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">image: url(:/pic.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">font: 36pt &quot;华文琥珀&quot;;</string>
          </property>
          <property name="text">
           <string>智慧小区</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;幼圆&quot;;</string>
            </property>
            <property name="text">
             <string>用户名：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_username"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;幼圆&quot;;</string>
            </property>
            <property name="text">
             <string>  密码：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_password"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="confirm_button">
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exit_button">
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>exit_button</sender>
   <signal>clicked()</signal>
   <receiver>Login</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
